crm_lead_project/models/crm_lead.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from lxml import etree
+from odoo import models, fields, api, _
+
+
+class CrmLead(models.Model):
+
+    _inherit = 'crm.lead'
+
+    project_id = fields.Many2one('project.project')
+
+    @api.multi
+    def action_open_project_form(self):
+        return {
+            'name': _('Project'),
+            'type': 'ir.actions.act_window',
+            'res_model': 'project.project',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'view_id': self.env.ref('project.edit_project').id,
+            'res_id': self.project_id.id,
+            'context': "{'remove_create_button': True}"
+        }
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
```

Comment:
Because we would then need glue modules to display the fields brought in by hr.timesheet or project.forecast on the project form, if those are installed.
This way, we ensure the opened form view contains all the things from its inheritances.